network/src/application/interface.rs

```diff
@@ -81,61 +81,6 @@ pub trait NetworkInterface<
     ) -> Result<(), PeerError>;
 }
 
-#[derive(Clone)]
```

Comment:
we added it in the previous commit to delete it in this one?

Reply:
Yeah, let me remove it from the history altogether.

I wanted to have a clean "send to one, and send to many system". Then realized this `SingleNetworkSender` was just more boilerplate than necessary.